README.md

```diff
@@ -1 +1,47 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] [OPTIONAL] Kindly consider using more descriptive commit messages, this will give you an idea of what changes are in each commit